website/docs/docs/use-dbt-semantic-layer/sl-cache.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Caching in the dbt Semantic Layer"
+id: "sl-cache"
+description: "Cache common queries to speed up performance and reduce query computation."
+tags: [Semantic Layer]
+sidebar_label: "Cache common queries"
+---
+
+
+The dbt Semantic Layer allows you to cache common queries in order to speed up performance and reduce compute on expensive queries. 
+
+There are two layers of caching:
+
+- [**Result caching**](#result-caching) leverages your data platform's built-in caching layer.
+- [**Declarative caching**](#declarative-caching) allows you to pre-warm the cache using saved queries.
+
+## Result caching
+
+Result caching leverages your data platform’s built-in caching layer and features. [MetricFlow](/docs/build/about-metricflow) generates the same SQL for multiple query requests, this means it can take advantage of your data platform’s cache.
+
+Here's how caching works, using Snowflake as an example, and should be similar across other data platforms:
+
+1. **Run from cold cache** &mdash; When a user runs a semantic layer query from their BI tool that hasn't been executed in the past 24 hours, the query scans the entire dataset and doesn't use the cache.
+2. **Run from warm cache** &mdash; If a user reruns the same query after 1 hour, the SQL generated and executed on Snowflake remains the same. On Snowflake, the result cache is set per user for 24 hours, which allows the repeated query to use the cache and return results faster.
+
+Different data platforms might have different caching layer and cache invalidation rules. Here's a list of resources on how caching works on some common data platforms:
+
+- [BigQuery](https://cloud.google.com/bigquery/docs/cached-results)
+- [DataBricks](https://docs.databricks.com/en/optimizations/disk-cache.html)
+- [Redshift](https://docs.aws.amazon.com/redshift/latest/dg/c_challenges_achieving_high_performance_queries.html#result-caching)
+- [Snowflake](https://community.snowflake.com/s/article/Caching-in-the-Snowflake-Cloud-Data-Platform)
+
+## Declarative caching
+
+Declarative Caching allows users to pre-warm the cache using Saved Queries by setting the cache enabled config to `true.` This is useful for optimizing the performance of key dashboards or common ad-hoc query requests. When you run your Saved Query, the semantic layer will build a cached table from a Saved Query in the customer's data platform. Any query requests with the same inputs as the Saved Query will hit the cache and return much more quickly. We’ll automatically invalidate the cache when we detect fresh data in any of the models upstream of metrics in your cached table. We’ll rebuild the cache the next time your Saved Query is run. The following diagram illustrates what happens when we receive a query request:
+
```

Comment:
this might be just me but i'm not clear on how this section joins with the 'populate cache' section. can you use declarative caching without exports? or do you need exports?

Reply:
you need an export!